### ISSUE TITLE
Thread argument being multiplied

### ISSUE THREAD

**Issue by parkerpayne** [epi2me-labs/wf-human-variation#81]
### Ask away!

Sometimes, after a few consecutive runs, the thread requirement starts becoming larger despite the inputted argument not changing. This causes an error and stops the run. The error message says something along the lines of "Not enough threads available: 32 Required: 40". I can attempt to recreate the issue if it would help, but I was wondering if this was a known/expected behavior. I find that eventually I have to lower the thread argument to 3 to get it to consistently be under the available count after the required number increases.

**Comment by SamStudio8** [comment]
Hi @parkerpayne, can you open this as a bug report so I can understand whether you're seeing this behaviour on the CLI or from the EPI2ME application? There is no behaviour inside the workflow that will increase the number of threads based on attempts. The number of threads used for alignment is calculated as follows:
```
cpus {params.ubam_map_threads + params.ubam_sort_threads + params.ubam_bam2fq_threads}
```

**Comment by parkerpayne** [comment]
That was indeed the issue we were experiencing, but after looking a little more the 'available' threads are also being limited. It says in the help it is limited by the executor threads, which I found a reference to in a config file. What is the proper way to raise this limit? We have 96 threads available but the executor threads are limited to 16. I found the config file on one system and changed it, which seemed to work, but I feel like digging into a hidden folder to edit a config is probably not the intended solution. Thank you for all your help!
